spec/models/question_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe Question do
+  describe "validations" do
+    it do
+      should validate_presence_of(:body)
+      .with_message("Question can't be blank")
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.